web/app/js/components/MetricsTable.test.js

```diff
@@ -46,24 +46,15 @@ describe('Tests for <MetricsTable>', () => {
       enableFilter: true
     });
     const component = mount(routerWrap(MetricsTable, extraProps));
-
```

Comment:
@cynthia-sg, thanks for splitting this out into another PR! 👍As I'm reviewing this, I'm seeing some review noise, with removed line breaks, etc. that is making it hard to see what code changed. Could you take another look at the diff here before I start reviewing? Thank you!